bundles/org.openhab.binding.boschindego/src/main/java/org/openhab/binding/boschindego/internal/IndegoController.java

```diff
@@ -119,6 +121,7 @@ protected <T> T getRequest(String path, Class<? extends T> dtoClass)
         int status = 0;
         try {
             Request request = httpClient.newRequest(BASE_URL + path).method(HttpMethod.GET)
+                    .timeout(REQUEST_TIMEOUT_MS, TimeUnit.MICROSECONDS)
```

Comment:
This method is also used for **longpoll**, so it can't simply be changed to 10 seconds. See:
https://github.com/openhab/openhab-addons/blob/7fb9efc8850236e515e1fbab10eee867620b6b56/bundles/org.openhab.binding.boschindego/src/main/java/org/openhab/binding/boschindego/internal/IndegoDeviceController.java#L97-L109

and also #13182. You now triggered me to revisit this. 🙂

Reply:
Should i leave this out, so you can set something up as you proceed in #13182 ?

Reply:
Yes, **longpoll** and synchronous calls over cellular network to a sleeping robot complicates it for this binding. If you can revert changes for this binding, I'll make sure to also lower the default timeout where possible when going back to this topic.

Reply:
Removed it.

Reply:
Removed this binding from this PR.
Created for reference: https://github.com/openhab/openhab-addons/issues/15507